project/profiles/capacity4more/modules/c4m/user/c4m_user/includes/c4m_user.theme.inc

```diff
@@ -346,8 +346,7 @@ function theme_c4m_user_contact_link(array $field) {
 
   // User administrators should always have access to personal contact forms.
   if (user_access('administer users') || user_access('access user contact forms')) {
-    $destination = array('destination' => request_uri());
-    return '<i class="fa fa-envelope-o"></i> ' . l(t('Contact this person'), "user/{$account->uid}/contact", array('query' => $destination));
+    return '<i class="fa fa-envelope-o"></i> ' . l(t('Contact this person'), "user/{$account->uid}/contact", array('query' => drupal_get_destination()));
```

Comment:
@anvmn 
I remember there was a reason why we didn't use `drupal_get_destination()` at first place